bundler/spec/dependabot/bundler/file_parser_spec.rb

```diff
@@ -526,6 +526,26 @@
               }]
             )
         end
+
+        context "with a gemspec with a float version number" do
```

Comment:
Can we have a spec for the `GemspecSanitizer` itself?

I also think this description needs rewriting.